SIT.Manager/SIT.Manager.csproj

```diff
@@ -19,6 +19,10 @@
 		<AvaloniaResource Include="Assets\**" />
 		<Compile Include="..\SIT.Manager.Desktop\AssemblyInfo.cs" Link="AssemblyInfo.cs" />
 		<EmbeddedResource Include="Localization\*.axaml" />
+		<AvaloniaXaml Remove="Classes\**" />
+		<Compile Remove="Classes\**" />
+		<EmbeddedResource Remove="Classes\**" />
+		<None Remove="Classes\**" />
```

Comment:
These lines can be removed, its a little pet peeve of mine is how happy the csproj files are to add stuff which is completely useless and makes them so messy.